recipe/meta.yaml

```diff
@@ -82,6 +87,15 @@ requirements:
 # Remove it with a new scikit-image release >=0.25.0.
 {% set tests_to_skip = tests_to_skip + " or test_thresholds_dask_compatibility" %}
 
+# see https://github.com/scipy/scipy/issues/20200 and https://github.com/scipy/scipy/issues/7408 
+{% set tests_to_skip = tests_to_skip + " or test_2d_bf[float16] or test_2d_cg[float16] or test_2d_cg_mg[float16] or test_2d_cg_j[float16]" %}
+
+# tifffile uses deprecated attribute `ndarray.newbyteorder` (for py313)
+{% set tests_to_skip = tests_to_skip + " or test_kidney_3d_multichannel or test_lily_multichannel" %}  # [py>=313]
```

Comment:
tifffile has a more recent release: https://github.com/cgohlke/tifffile/compare/v2024.12.12...v2025.1.10
(In case it helps.)

Reply:
The latest version improves only type hints and drops Python 3.10. I think it won't help